<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>RecvBuffer</string>
    </property>
    <widget class="QPlainTextEdit" name="recvTextBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>SendBuffer</string>
    </property>
    <widget class="QPlainTextEdit" name="sendTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="SendBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPortBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenPort</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portNameComboBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionSelect_COM"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelect_COM">
   <property name="text">
    <string>Select COM</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
